cicecore/cicedyn/general/ice_init.F90

```diff
@@ -1569,33 +1571,32 @@ subroutine input_data
          abort_list = trim(abort_list)//":22"
       endif
 
-      if(histfreq_base /= 'init' .and. histfreq_base /= 'zero') then
-         write (nu_diag,*) subname//' ERROR: bad value for histfreq_base, allowed values: init, zero'
-         abort_list = trim(abort_list)//":24"
-      endif
+      do n = 1,max_nstrm
+         if(histfreq_base(n) /= 'init' .and. histfreq_base(n) /= 'zero') then
+            write (nu_diag,*) subname//' ERROR: bad value for histfreq_base, allowed values: init, zero:'//trim(histfreq_base(n))
+            abort_list = trim(abort_list)//":24"
+         endif
+
+         if(dumpfreq_base(n) /= 'init' .and. dumpfreq_base(n) /= 'zero') then
+            write (nu_diag,*) subname//' ERROR: bad value for dumpfreq_base, allowed values: init, zero:'//trim(dumpfreq_base(n))
+            abort_list = trim(abort_list)//":25"
+         endif
+
+         if (.not.(scan(dumpfreq(n)(1:1),'ymdhx1YMDHX') == 1 .and. (dumpfreq(n)(2:2) == '1' .or. dumpfreq(n)(2:2) == ' '))) then
+            if (my_task == master_task) then
+               write(nu_diag,*) subname//' WARNING: unrecognized dumpfreq=', trim(dumpfreq(n))
+               write(nu_diag,*) subname//' WARNING:   No restarts files will be written for this stream'
+               write(nu_diag,*) subname//' WARNING:   Allowed values : ymdhx1 followed by an optional 1'
```

Comment:
minor, but I think the previous message was more clear by separating each option instead of `ymdhx1` all together